src/Makefile-env.am

```diff
@@ -256,8 +256,6 @@ if ENABLE_COVERAGE
 EXTRALIBS += -lgcov
 endif # ENABLE_COVERAGE
 
-LIBCOMMON += -luuid
```

Comment:
So what is the source of your uuid routines then??  libc??
I tried that at the beginning, but it bommed out.

I will try that again, in a seperate run.,
Thanx for the pointer.